src/transport_ofi.c

```diff
@@ -546,6 +548,24 @@ static int populate_mr_tables(void)
     return 0;
 }
 
+static inline int atomicvalid_rtncheck(int ret, int atomic_size, long atomicwarn,
+                                    char strOP[], char strDT[]) {
+
+    if(ret != 0 || atomic_size == 0) {
+        fprintf(stderr, "%s OFI detected no support for atomic '%s'"
+               "on type '%s'\n", (atomicwarn ? "Warning" : "Error"),
+                strOP, strDT);
```

Comment:
this function just passes forward the predetermined ret value, if it comes in success it will return success